include/current.h

```diff
@@ -1,67 +1,61 @@
 
+ 
 /*
  * Content: class that measures current using the ACS711EX Hall Effect Sensor
  * Author : Forbes Choy
  * Team   : Sustaingineering UBC
  */
-
+ 
 #pragma once
 #include <Arduino.h>
 #include <farmSensor.h>
-
-
-class current
+ 
+ 
+class current: public farmSensor
 {
 private:
-    int pin; //analog pin number (the GPIO pin number) for data reading
+    int m_pin; //analog pin number (the GPIO pin number) for data reading
     float m_data; //changed to float instead of integer
```

Comment:
There is no need for `m_pin` and `m_data` here since they are declared in `protected` scope in `farmSensor`.